tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,88 @@
+# realistic, not spherical, channel positions, we will use a different montage
+# to demonstrate controlling how channels are projected to 2d space.
+
+biosemi_montage = mne.channels.make_standard_montage('biosemi64')
+biosemi_montage.plot(show_names=False)
+
+###############################################################################
+# By default a sphere  with an origin in ``(0, 0, 0)`` x, y, z coordinates and
+# radius of ``0.095`` meters (9.5 cm) is used. You can use a different sphere
+# radius by passing a single value to ``sphere`` argument in any function that
+# plots channels in 2d (like :meth:`~mne.channels.DigMontage.plot`that we use
+# here, but also for example :func:`mne.viz.plot_topomap`):
+
+biosemi_montage.plot(show_names=False, sphere=0.07)
+
+###############################################################################
+# To control not only radius, but also the sphere origin, pass a
+# ``(x, y, z, radius)`` tuple to ``sphere`` argument:
+
+biosemi_montage.plot(show_names=False, sphere=(0.03, 0.02, 0.01, 0.075))
+
+###############################################################################
+# In mne-python the head center and therefore the sphere center are calculated
+# using fiducial points. Because of this the head circle represents head
+# circumference at the nasion and ear level, and not where it is commonly
+# measured in 10-20 EEG system: above nasion at T4/T8, T3/T7, Oz, Fz level.
+# Notice below that by default T7 and Oz channels are placed within the head
+# circle, not on the head outline:
+
+biosemi_montage.plot()
+
+###############################################################################
+# If you have previous EEGLAB experience you may prefer its convention to
+# represent 10-20 head circumference with the head circle. To get EEGLAB-like
+# channel layout you would have to move the sphere origin a few centimeters
+# up on the z dimension:
+
+biosemi_montage.plot(sphere=(0, 0, 0.035, 0.094))
+
+###############################################################################
+# Instead of approximating the EEGLAB-esque sphere location as above, you can
+# calculate the sphere origin from position of Oz, Fpz, T3/T7 or T4/T8
+# channels. This is easier once the montage has been applied to the data and
+# channel positions are in head space - see this example
+# (all the code below should be a separate example):
+
+n_channels = len(biosemi_montage.ch_names)
+fake_info = mne.create_info(ch_names=biosemi_montage.ch_names, sfreq=250.,
+                            ch_types=['eeg'] * n_channels)
+data = np.random.random((n_channels, 10))
+fake_raw = mne.io.RawArray(data, fake_info)
+fake_raw.set_montage(biosemi_montage)
+
+check_ch = ['Oz', 'Fpz', 'T7', 'T8']
+pos = fake_raw._get_channel_positions(picks=check_ch)
+radius = pos[-1, 0].mean()
+x, y, z = pos[0, 0], pos[-1, 1], pos[:, -1].mean()
+
+with np.printoptions(precision=5, suppress=True):
+    print(np.array([x, y, z, radius]))
```

Comment:
Personally I would just do this, but either way is fine
```suggestion
print([f'{v:0.5f} for v in [x, y, z, radius]])
```

Reply:
right, I thought the formatting string appraoch would be a bit convoluted for tutorial/example but my solution with context manager is probably equally opaque. :)

Reply:
Agreed neither is great, whichever one is fine by me